prov/rxm/src/rxm_conn.c

```diff
@@ -1089,14 +1093,39 @@ static int rxm_conn_handle_notify(struct fi_eq_entry *eq_entry)
 		assert(handle->state == RXM_CMAP_SHUTDOWN);
 		FI_DBG(&rxm_prov, FI_LOG_EP_CTRL, "freeing handle: %p\n", handle);
 		cmap = handle->cmap;
+		rxm_ep = container_of(cmap->ep, struct rxm_ep, util_ep);
+
+		rxm_conn_free(handle);
```

Comment:
I think this needs to be freed after flushing the CQ.

Reply:
`rxm_conn_free` is actually what was closing the ep. I guess I should `rxm_conn_close` here and move this free to after the flush